service/build.gradle

```diff
@@ -236,14 +236,7 @@ sonarqube {
 repositories {
   mavenLocal()
   mavenCentral()
-  // TODO: that's the only dependency to move (confirm with ops when that's migrated) https://tools.hmcts.net/jira/browse/FPLA-2978
-  maven {
-    url "https://dl.bintray.com/hmcts/hmcts-maven"
-  }
-  //maven {
-  //  url "https://jitpack.io"
-  //}
-
+  jcenter()
```

Comment:
@samuelzerozero @tomaszpowroznik I thought we removed jcenter with its deprecation

Reply:
true, but i'm confused, as we still have 
```
maven {
    url "https://dl.bintray.com/hmcts/hmcts-maven"
  }
```
do we know where artefacts has been ported?

btw, jcenter will serve artefacts till 1st Feb 2022

Reply:
and i think that all of this works without jcenter only because artefacts are cached in our locals and in jenkins agents.  When i cleared my cache and build then plenty of artefacts are missing